8Knot/pages/affiliation/visualizations/gh_org_affiliation.py

```diff
@@ -46,63 +64,67 @@
                 ),
                 dcc.Loading(
                     dcc.Graph(id=f"{PAGE}-{VIZ_ID}"),
+                    style={"marginBottom": "1rem"},
+                ),
+                html.Hr(  # Divider between graph and controls
+                    style={
+                        "borderColor": "#909090",
+                        "margin": "1.5rem -1.5rem",
+                        "width": "calc(100% + 3rem)",
+                    }
```

Comment:
a divider may not be worth moving into a shared visualization base too, but that might be another helpful thing for consistency/reuse